src/apis/contracts_api.ts

```diff
@@ -364,7 +365,13 @@ export class ContractsAPI {
     public async getTokenSymbolByIndexAsync(index: BigNumber): Promise<string> {
         const tokenRegistryContract = await this.loadTokenRegistry();
 
-        return tokenRegistryContract.getTokenSymbolByIndex.callAsync(index);
+        const symbol = await tokenRegistryContract.getTokenSymbolByIndex.callAsync(index);
+
+        if (symbol === "") {
```

Comment:
any case where this comes back undefined or null? because those wouldn't throw.

Reply:
I'll add those cases just in case.